src/devices/sound/ap2010pcm.h

```diff
@@ -10,12 +10,13 @@
 
 **********************************************************************/
 
-#ifndef MAMESOUND_AP2010PCM_H
-#define MAMESOUND_AP2010PCM_H
+#ifndef MAME_SOUND_AP2010PCM_H
+#define MAME_SOUND_AP2010PCM_H
 
 #pragma once
 
-#include <queue>
+// FIXME: Games check this against 0x1ff, but samples are lost with that limit
+#define FIFO_MAX_SIZE 0x800
```

Comment:
Please turn this into a private class member so it doesn’t pollute the global namespace, like:
```c++
	static inline constexpr FIFO_MAX_SIZE = 0x800;
```
inside the private section of the class.